sonar-plugin-api/src/main/java/org/sonar/api/ce/measure/test/TestIssue.java

```diff
@@ -127,8 +128,16 @@ public Builder setRuleKey(RuleKey ruleKey) {
       return this;
     }
 
+    /**
+     * @deprecated since 5.5, use {@link #setEffort(Duration)} instead
+     */
+    @Deprecated
     public Builder setDebt(@Nullable Duration debt) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)